wctl/ws_accounts.go

```diff
@@ -113,22 +113,3 @@ func parseAccountStakeUpdated(c *Client, v *fastjson.Value) error {
 	}
 	return nil
 }
-
-func parseAccountRewardUpdated(c *Client, v *fastjson.Value) error {
```

Comment:
Should we figure out why this is no longer used ?

Reply:
do you know where it was used previously?

Reply:
I'm not sure -- it may have never been used.  We don't have to add it back but we should create an issue to track down whether or not it should be used.

Reply:
There's a mistake in PollAccounts that has gone undetected. 

We're suppose to use it to parse "reward_updated" event.

https://github.com/perlin-network/wavelet/blob/d27bb3167819201e17375c1dbfe7a9276c86a789/wctl/ws_accounts.go#L26-L27.

Reply:
Excellent find !